Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM ubuntu:latest
+
+ENV TZ=Europe/Kiev
+
+ARG DEBIAN_FRONTEND=noninteractive
+
+RUN apt-get update && apt-get install -y \
+    cmake \
+    g++ \
+    build-essential \
+    gperf \
+    openssl \
+    libssl-dev \
+    zlib1g-dev \
+    git \
+    && rm -rf /var/lib/apt/lists/*
+
+
+RUN git clone --recursive https://github.com/tdlib/telegram-bot-api.git && \
+    cd telegram-bot-api/ && \
+    mkdir build && \
+    cd build
```

Comment:
I don't know if it's a serious issue, but these two lines are redundant. The next line will do this anyway